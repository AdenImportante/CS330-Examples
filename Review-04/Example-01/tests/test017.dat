#! /bin/sh

# Test entire program with input set 02
# disregard formatting (spaces, newlines, and letter case)

./storage items-02.txt 11 | sed 's/[ ]//g' | grep -vE "^$" | tr '[:upper:]' '[:lower:]'