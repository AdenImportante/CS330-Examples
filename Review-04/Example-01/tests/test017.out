processinglog:
(s)pickaxe
(s)potatocamera
(s)shovel
(s)pickaxe
(s)speed-ii-potion
(s)tomato
(s)potatocamera
(s)axe
(s)boots
(s)boots
(s)chestplate
(s)speed-ii-potion
playerstoragesummary:
-used91%of11slots
nme:pickaxe
dur:100
spd:1
mtl:diamond
mdr:fortune(lvl5)
nme:potatocamera
eft:imagequality-97%
use:5
qty:2
nme:shovel
dur:20
spd:3
mtl:gold
mdr:unbreaking(lvl2)
nme:pickaxe
dur:100
spd:1
mtl:diamond
mdr:fortune(lvl5)
nme:speed-ii-potion
eft:spd*2
use:1
qty:2
nme:tomato
eft:hunger-10
use:2
qty:1
nme:axe
dur:10
spd:2
mtl:stone
mdr:unbreaking(lvl2)
nme:boots
dur:100
def:10
mtl:diamond
mdr:protection(lvl3)
emt:lightning
nme:boots
dur:100
def:10
mtl:diamond
mdr:featherfalling(lvl4)
emt:lightning
nme:chestplate
dur:100
def:10
mtl:diamond
mdr:defense(lvl9)
emt:light
